meta-resin-common/recipes-containers/docker/docker_git.bb

```diff
@@ -0,0 +1,137 @@
+HOMEPAGE = "http://www.docker.com"
+SUMMARY = "Linux container runtime"
+DESCRIPTION = "Linux container runtime \
+ Docker complements kernel namespacing with a high-level API which \
+ operates at the process level. It runs unix processes with strong \
+ guarantees of isolation and repeatability across servers. \
+ . \
+ Docker is a great building block for automating distributed systems: \
+ large-scale web deployments, database clusters, continuous deployment \
+ systems, private PaaS, service-oriented architectures, etc. \
+ . \
+ This package contains the daemon and client. Using docker.io on non-amd64 \
+ hosts is not supported at this time. Please be careful when using it \
+ on anything besides amd64. \
+ . \
+ Also, note that kernel version 3.8 or above is required for proper \
+ operation of the daemon process, and that any lower versions may have \
+ subtle and/or glaring issues. \
+ "
+
+SRCREV = "20f81dde9bd97c86b2d0e33bbbf1388018611929"
+SRCBRANCH = "v1.10.3"
+SRC_URI = "\
+	git://github.com/docker/docker.git;branch=${SRCBRANCH};nobranch=1 \
+	file://docker.service \
+	"
+
+# Apache-2.0 for docker
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=aadc30f9c14d876ded7bedc0afd2d3d7"
+
+S = "${WORKDIR}/git"
+
+DOCKER_VERSION = "1.10.3"
+PV = "${DOCKER_VERSION}+git${SRCREV}"
+
+DEPENDS = " \
+    go-cross \
+    btrfs-tools \
+    "
+
+DEPENDS_append_class-target = "lvm2"
+RDEPENDS_${PN} = "curl git util-linux iptables"
```

Comment:
This come from meta-vrit. I believed it was needed during build time. Did not give it much thought.